TESTS/mbedmicro-rtos-mbed/signals/main.cpp

```diff
@@ -55,6 +55,11 @@ struct Sync {
 void error(const char* format, ...) {
     (void) format;
 }
+
+mbed_error_status_t mbed_error(mbed_error_status_t error_status, const char *error_msg, unsigned int error_value, const char *filename, int line_number) 
```

Comment:
Do we need this code here?

Reply:
Can you please explain it? Do you mean we can have it under macro?

Reply:
My comment was inline to previous one, till we replace `error` call from CMSIS/RTX lib, we should not add this code. Its should be part of same commit, to help in keeping track of CMSIS changes for updates.